_docs-2-0/administration/in-depth-install.md

```diff
@@ -294,27 +294,11 @@ will expect the KeyStore in the same location.
 
 ### Client Configuration
 
-In version 1.6.0, Accumulo included a new type of configuration file known as a client
-configuration file. One problem with the traditional "site.xml" file that is prevalent
-through Hadoop is that it is a single file used by both clients and servers. This makes
-it very difficult to protect secrets that are only meant for the server processes while
-allowing the clients to connect to the servers.
+Accumulo clients are configured in a different way than Accumulo servers. Clients are
+configured when [an Accumulo Connnector is created][client-conn] using Java builder methods
+or a `accumulo-client.properties` file containing [client properties][client-props].
 
-The client configuration file is a subset of the information stored in accumulo-site.xml
-meant only for consumption by clients of Accumulo. By default, Accumulo checks a number
-of locations for a client configuration by default:
-
-* `/path/to/accumulo/conf/client.conf`
```

Comment:
If using the new API, will it still look in these locations?

Reply:
No, the new API doesn't look in those locations.